sdk/src/main/java/com/sygic/travel/sdk/auth/service/AuthServiceUrlHolder.kt

```diff
@@ -0,0 +1,13 @@
+package com.sygic.travel.sdk.auth.service
+
+object AuthServiceUrlHolder {
```

Comment:
This is definitely not a good solution. 
- Object imo this is wrong
- there should be interface
- with default static implementation

Also, not sure if the name of the class should be so specific.  Also, it is not about auth url.
I have told you it should configure both urls!.